config/mailgun-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  domain: ${DOMAIN}
+  domainAPIKey: ${DOMAIN_API_KEY}
+kind: Secret
+metadata:
+  name: mailgun-secret
+  namespace: toolchain-host-operator
```

Comment:
Do not hard code the namespace. Use `${NAMESPACE}` instead.

Reply:
@alexeykazakov is this because it's a secret? i noticed that we hard-code it in other places such as here: https://github.com/codeready-toolchain/toolchain-infra/blob/master/config/reg_service_route.yaml#L5 so i thought it was fine..

Reply:
Yeah.. https://github.com/codeready-toolchain/toolchain-infra/blob/master/config/reg_service_route.yaml#L5 is the only yaml where we have `toolchain-host-operator` hardcoded. I would rather fix it there and wouldn't have it hardcoded at all. It would be easier to parametrize the namespace name later if we need it.
But it's a minor think anyway. We could keep it hardcoded for now.